install.md

```diff
@@ -173,7 +173,7 @@ In Ubuntu zesty and xenial, you can use these commands:
   apt-add-repository -y ppa:projectatomic/ppa
   apt-get -y -qq update
   apt-get -y install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libostree-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
-  apt-get -y install golang-1.8
+  apt-get -y install golang-1.10
```

Comment:
Do those versions of Ubuntu offer go 1.10?

Reply:
I hopped on the latest Ubuntu server that I could find and it had 1.10, I'm not sure about zesty and xenial in particular.  For that matter our install instructions for Ubuntu seem to be off atm, I was able to get golang 1.10, but compilations were failing.